src/transformers/modeling_tf_t5.py

```diff
@@ -942,37 +989,58 @@ def call(self, inputs, **kwargs):
         >>> last_hidden_states = outputs[0]  # The last hidden-state is the first element of the output tuple
 
         """
-
-        if isinstance(inputs, dict):
-            kwargs.update(inputs)
+        if isinstance(inputs, (tuple, list)):
```

Comment:
There were was one inconsistency in TF T5 before in that the variable `input_ids` was wrongly called `inputs` @sshleifer .
Also TF T5 is made completely keras compilation compatible here which was not the case before.